### PR TITLE
[HER-157] Update the design of navbar

### PR DESCRIPTION
**Pull request by andreea-dumitrascu**
## Description
We should update the design of the navbar as it is in the links: [Desktop](https://xd.adobe.com/view/f1281b54-d458-42c1-b28b-7dcca2c23b31-a47b/specs/?fbclid=IwAR3x9zSAwHJJIrQaGxIQvAKjAwys50dFidZfz7ZnOpH4-rzQX-W-kQO4LVo), [Tablet](https://xd.adobe.com/view/c08e8b21-5cdf-469b-9e66-2c500a5ec043-95dd/?fbclid=IwAR1x4Wn89kjsO0ZivtrxqBztZvh67ubeCMJGVc8eMmivnjwBRwteIr7zP20) ,[Phone](https://xd.adobe.com/view/c5fc8477-17c8-4014-872e-6e6575603d84-90af/?fbclid=IwAR2aCw3az4LwYWh4oYOW08rfN_9r8F8xh8u98QghtsQUYmHNQoKX8QUCAt4).

## Related
[HER-157](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-157)

### CODE REVIEW

File: src/core/layout/navbar/links/hooks/use-cameras-props.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { useStyles } from 'core/layout/navbar/links/hooks/use-styles'
+
+export const useCamerasProps = () => ({
```
Comment (miutamihai) [inline src/core/layout/navbar/links/hooks/use-cameras-props.js]: image
All these hooks are __really__ similar. Can't we avoid all this code duplication somehow?

File: src/core/layout/navbar/links/hooks/use-on-page.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { makeStyles } from '@material-ui/core'
+
+const root = {
+  color: '#1A856B',
```
Comment (miutamihai) [inline src/core/layout/navbar/links/hooks/use-on-page.js]: Let's declare all these colors as variables in our theme.

Check their documentation on how to do that [here](https://mui.com/customization/theming/)

File: src/core/layout/navbar/links/hooks/use-styles.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useLocation } from 'react-router-dom'
+
+import { useOnPage } from 'core/layout/navbar/links/hooks/use-on-page'
+import { useOutsidePage } from 'core/layout/navbar/links/hooks/use-outside-page'
+
+export const useStyles = path => {
+  const location = useLocation()
+  const onPage = useOnPage()
```
Comment (miutamihai) [inline src/core/layout/navbar/links/hooks/use-styles.js]: These namings are ...weird. Let's use something like `activeStyle` and `inactiveStyle`.

File: src/core/layout/navbar/links/line.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+import PropTypes from 'prop-types'
+
+const onPage = { backgroundColor: '#1A856B', height: 1, marginLeft: 4, width: '100%' }
```
Comment (miutamihai) [inline src/core/layout/navbar/links/line.jsx]: Same here.

File: src/core/layout/navbar/links/line.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+const outsidePage = { display: 'none' }
+
+export const Line = ({ path }) => {
```
Comment (miutamihai) [inline src/core/layout/navbar/links/line.jsx]: Actually, why create a separate component for the line under when we can just apply css for that?

File: src/core/layout/navbar/logo/logo-desktop.jsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import * as Material from '@material-ui/core'
+
+import Owl from 'core/layout/navbar/logo/icons8-owl-100.png'
+import { useStyles } from 'core/layout/navbar/logo/styles'
+import { useNavigation } from 'core/layout/navbar/links/hooks/use-navigation'
+
+export const LogoDesktop = () => <Material.Grid item>
```
Comment (miutamihai) [inline src/core/layout/navbar/logo/logo-desktop.jsx]: image
These two components are identical... Do we really need to duplicate them?

**Review by miutamihai — CHANGES_REQUESTED** [review]
image
image
The font weight used here does not align with the design. Let's rectify that.

image
Also, this probably not the expected behaviour here. Let's ditch that sticky appbar.

File: src/core/layout/navbar/links/hooks/use-cameras-props.js (hunk 1 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/links/hooks/use-cameras-props.js]: Done.

File: src/core/layout/navbar/links/hooks/use-styles.js (hunk 3 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/links/hooks/use-styles.js]: Done

File: src/core/layout/navbar/links/hooks/use-on-page.js (hunk 2 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/links/hooks/use-on-page.js]: Done

File: src/core/layout/navbar/logo/logo-desktop.jsx (hunk 6 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/logo/logo-desktop.jsx]: Done

File: src/core/layout/navbar/links/line.jsx (hunk 4 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/links/line.jsx]: Done

File: src/core/layout/navbar/links/line.jsx (hunk 5 above)
Comment (andreea-dumitrascu) [reply src/core/layout/navbar/links/line.jsx]: Done

File: src/core/layout/navbar/logo/hooks/use-img-props.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import Owl from 'core/layout/navbar/logo/icons8-owl-100.png'
+import { useNavigation } from 'core/layout/navbar/links/hooks/use-navigation'
+import { useClassName } from 'core/layout/navbar/logo/hooks/use-class-name'
+
+export const useImgProps = () => ({
+  ...useClassName(),
```
Comment (mihairadulescu) [inline src/core/layout/navbar/logo/hooks/use-img-props.js:6]: className:useClassName() would  be better,.

**Comment by mihairadulescu** [comment]
add 

`
    position: sticky;
    background: white;
 `
 
     
image

**Comment by mihairadulescu** [comment]
add 

```
    position: sticky;
    background: white;
    ```
    
    and it should be like
    
    
image

**Comment by andreea-dumitrascu** [comment]
> background: white

Done.

**Review by mihairadulescu — APPROVED** [review]

**Comment by mihairadulescu** [comment]
Cool! Redeploying.

**Review by miutamihai — APPROVED** [review]

**Pull request merged**